docs/ai_actions/ai_actions_guide.md

```diff
@@ -0,0 +1,143 @@
+The service connector wraps all data into a prompt or another suitable format and sends it to an external AI service.
+
+When the AI Service returns a response, the response goes back through the service connector and passes to the framework.
+It can then be presented to the user in any way necessary.
+
+### Core concepts
+
+#### AI service
+
+AI service is a third party platform that provides access to artificial intelligence tools and capabilities.
+It executes tasks that it receives through a service connector.
+
+#### Action
+
+Actions are tasks or functions that the AI service is set to perform.
+Each action is a combination of an AI action type and an AI action configuration.
+Action types define what kind of task the AI service performs, while AI action configurations specify how the task should be executed.
+This clear separation allows for a flexible system where actions can be created, managed, and customized with minimal effort.
+
+#### AI action type
+
+AI action types are templates predefined by developers.
+Each AI action type defines the structure and nature of the task that the AI service performs.
+AI action types could be designed, for example, to generate text based on an image, translate a passage of text, or generate a video clip based on a description.
+By defining AI action types, developers can create a wide range of functionalities that can be deployed within the application.
+
+#### AI action configuration
+
+AI action configurations store detailed parameters needed to generate AI actions based on AI action types.
+Website administrators manage AI action configurations in the [**Admin** panel](admin_panel.md), where they customize and fine-tune the behavior of each AI action.
+It might involve setting specific parameters used by the AI service, a response length, an expense limit, or configuring how the output should be handled.
+By making such adjustments, administrators can ensure that the actions are tailored to meet the needs of your organization.
+
+#### Model
+
+Once an AI action is defined and configured, it must be executed, and this is where models come into play.
+Each model is designed to work with a specific AI service and AI action type pair.
+Pieces of PHP code that are responsible for resolving a model are called handlers.
+They may include hardcoded prompts for conversational AI services like ChatGPT, or operate without prompts in the case of other types of AI.
+Handlers take parameters defined in the AI action type and configuration, combine it with user input and any predefined settings or prompts, and pass this information to the AI service for processing.
+
+### Triggering actions from the UI
+
+Among other elements, AI Actions LTS update includes UI components for:
+
+- AI action management in the **Admin** panel
+- text modification in online editor
+- alt-text generation feature in the image management modal
```

Comment:
The UI component isn't really _for_ specific actions, it's meant for all of them. Custom action types should be able to benefit from it as well (unless there are limitations @micszo ?)

Reply:
I reworded it slightly to say that these components are used in ...
It does not mean they are exclusively intended for this use, they could be adapted for other uses.